src/main/java/Command.java

```diff
@@ -0,0 +1,17 @@
+public abstract class Command {
+    /**
+     * Run the command on console.
+     * @param tasks Task list
+     * @param storage Handle file storage
+     * @param ui Show interaction
+     */
+
```

Comment:
Empty line between description and parameter section.
Punctuation behind each parameter description.
No blank line between the documentation block and the method/class.